modules/heat_conduction/src/meshgenerators/PatchSidesetGenerator.C

```diff
@@ -230,6 +250,97 @@ PatchSidesetGenerator::generate()
   return mesh;
 }
 
+void
+PatchSidesetGenerator::partition(MeshBase & mesh)
+{
+  if (_partitioner_name == "grid")
+  {
+    // Figure out the physical bounds of the given mesh
+    auto bounding_box = MeshTools::create_bounding_box(mesh);
+    const auto & min = bounding_box.min();
+    const auto & max = bounding_box.max();
+    const auto & delta = max - min;
+
+    // set number of elements
+    unsigned int nx, ny;
+    if (_dim == 1)
+    {
+      nx = _n_patches;
+      ny = 1;
+    }
+    else
+    {
+      nx = std::round(std::sqrt(delta(0) / delta(1) * _n_patches));
+      ny = std::round(std::sqrt(delta(1) / delta(0) * _n_patches));
+      if (_n_patches != nx * ny)
+      {
+        _console << "Note: For creating radiation patches for boundary " << _sideset
+                 << " using grid partitioner number of patches was changed from " << _n_patches
+                 << " to " << nx * ny << std::endl;
+        _n_patches = nx * ny;
+      }
+    }
+
+    Real dx = delta(0) / nx;
+    Real dy = delta(1) / ny;
```

Comment:
I know these are just temporaries, but the 'const' indicates that they don't change later. I just like that from a readability standpoint, you can take it or leave it. Just my 2 cents. There are a few others you could probably do this with if you want.
```suggestion
   const Real dx = delta(0) / nx;
   const Real dy = delta(1) / ny;
```